tensorflow/lite/micro/testing/robot.resource

```diff
@@ -0,0 +1,26 @@
+*** Variables ***
```

Comment:
this file is failing internal checks because it does not have a license on top.

Please rename to robot.resource.txt and that (the .txt extension) should exclude it needing a license since adding the license here doesn't really work.

Reply:
Renamed as recommended